src/sentry/static/sentry/app/views/organizationDashboard/projectCard.jsx

```diff
@@ -37,40 +59,83 @@ class ProjectCard extends React.Component {
-      <StyledProjectCard>
-        <Flex justify="space-between" align="center">
-          <StyledLink to={`/${params.orgId}/${project.slug}/`}>
-            <strong>{project.slug}</strong>
-          </StyledLink>
-          <Tooltip title={bookmarkText}>
-            <Star
-              active={project.isBookmarked}
-              className="project-select-bookmark icon icon-star-solid"
-              onClick={this.toggleProjectBookmark}
-            />
-          </Tooltip>
-        </Flex>
-        <ChartContainer>
-          {!stats ? (
-            <NoStats>{t('Project stats are not available at the moment.')}</NoStats>
-          ) : (
-            <Chart stats={stats} noEvents={!project.firstEvent} />
-          )}
-          {!project.firstEvent && <NoEvents />}
-        </ChartContainer>
-        <PlatformList project={project} orgId={params.orgId} />
-      </StyledProjectCard>
+      <ProjectCardWrapper
+        data-test-id={project.slug}
+        width={['100%', '50%', '33%', '25%']}
+      >
+        {stats === null && <LoadingCard />}
+        {stats && (
+          <StyledProjectCard>
+            <Flex justify="space-between" align="center">
+              <StyledLink to={`/${params.orgId}/${project.slug}/`}>
+                <strong>{project.slug}</strong>
+              </StyledLink>
+              <Tooltip title={bookmarkText}>
+                <Star
+                  active={project.isBookmarked}
+                  className="project-select-bookmark icon icon-star-solid"
+                  onClick={this.toggleProjectBookmark}
+                />
+              </Tooltip>
+            </Flex>
+            <ChartContainer>
+              {!stats ? (
+                <NoStats>{t('Project stats are not available at the moment.')}</NoStats>
+              ) : (
+                <Chart stats={stats} noEvents={!project.firstEvent} />
+              )}
+              {!project.firstEvent && <NoEvents />}
+            </ChartContainer>
+            <PlatformList project={projectDetails} orgId={params.orgId} />
+          </StyledProjectCard>
+        )}
+      </ProjectCardWrapper>
     );
   }
 }
 
+const ProjectCardContainer = createReactClass({
+  propTypes: {
+    project: SentryTypes.Project,
+  },
+  mixins: [Reflux.listenTo(ProjectsStatsStore, 'onProjectStoreUpdate')],
+  getInitialState() {
+    const {project} = this.props;
+    const initialState = ProjectsStatsStore.getInitialState() || {};
+    return {
+      projectDetails: initialState[project.id] || null,
+    };
+  },
+  onProjectStoreUpdate(itemsById) {
+    const {project} = this.props;
+
+    // Don't update state if we already have stats
+    if (!!this.state.projectDetails) return;
+    if (!itemsById[project.id]) return;
+
+    this.setState({
+      projectDetails: itemsById[project.id],
+    });
+  },
+  render() {
+    const {projectDetails} = this.state;
+    return (
+      <ProjectCard
+        {...this.props}
+        projectDetails={projectDetails}
```

Comment:
If i'm not mistaken isn't `projectDetails` essentially just `project`? If so, why do we need to pass both to `<ProjectCard />`?